test/lazy/test_sum_kronecker_lazy_tensor.py

```diff
@@ -47,3 +47,19 @@ def evaluate_lazy_tensor(self, lazy_tensor):
             lazy_tensor.lazy_tensors[1].lazy_tensors[0].tensor, lazy_tensor.lazy_tensors[1].lazy_tensors[1].tensor
         )
         return res1 + res2
+
+    def test_inv_quad_logdet(self):
+        # mock call cg here
+        self.__class__.should_call_cg = True
+        super().test_inv_quad_logdet()
+        self.__class__.should_call_cg = False
```

Comment:
Might this lead to issues with test concurrency if you manually mock this here?  How about the following (also below)?
```suggestion

    @mock.patch(
        "gpytorch.lazy.test_sum_kronecker_lazy_tensor.should_call_cg",
        new_callable=PropertyMock, 
        return_value=True,
    ):
    def test_inv_quad_logdet(self):
        super().test_inv_quad_logdet()
```

Reply:
This approach sadly doesn't quite work, will deal with concurrency issues if they exist later on.